spec/timescaledb/connection_spec.rb

```diff
@@ -28,14 +28,14 @@
       let(:sql) { "select 1 as one" }
```

Comment:
/bug this is performing linting and is likely to conflict to the project lint settings - therefore becomes noise. Maybe if there's a way to parse the lint settings it could help, but then the linter itself is supposed to be doing that.